.travis.yml

```diff
@@ -1,9 +1,5 @@
 sudo: false
 language: node_js
 node_js:
-  - 0.12
```

Comment:
because yarn make tests fail: `Node.js version v0.10.48 does not meet requirement for yarn. Please use Node.js 4 or later.`